src/src/TemplateBundle/Controller/CustomTemplateController.php

```diff
@@ -77,14 +77,20 @@ public static function getLanguageNames($templateName)
         return array_map('basename', glob($languagesPath));
     }
 
-    public function getLanguageFile($templateName, $language, $context = [])
+    protected function getFileToYaml($path, $context)
```

Comment:
Nemyslim si, ze `getFileToYaml` je spravne meno pre metodu. Robi predsa to, ze z Yaml vytvori objekt (resp. php array) a ten vrati. Nebolo by to lepsie premenovat na `getArrayFromYaml` alebo nieco podobne?

Reply:
pravda, je to zle meno. opravim to.